pramen-py/tests/test_utils/test_file_system.py

```diff
@@ -53,11 +61,236 @@
         ),
         (
-            "//D:/Users/AB025F5/example_dependency_table",
-            "file://D:/Users/AB025F5/example_dependency_table",
+            "file://Users/AB025F5/example_dependency_table",
+            "file://Users/AB025F5/example_dependency_table",
+        ),
+        (
+            "//D:/Users/AB025F5//example_dependency_table//",
+            "file:///D:/Users/AB025F5/example_dependency_table",
+        ),
+        (
+            "file:\\\\\\D:\\Users\\AB025F5\\example_dependency_table\\",
+            "file:///D:/Users/AB025F5/example_dependency_table",
+        ),
+        (
+            "file:\\D:\\Users\\AB025F5\\example_dependency_table\\",
+            "file:///D:/Users/AB025F5/example_dependency_table",
+        ),
+        (
+            "file:\\D:\\Users\\AB025F5\\example_dependency_table\\",
+            "file:///D:/Users/AB025F5/example_dependency_table",
+        ),
+        (
+            "\\D:\\Users\\AB025F5\\example_dependency_table\\",
+            "file:///D:/Users/AB025F5/example_dependency_table",
+        ),
+        (
+            "D:\\Users\\AB025F5\\example_dependency_table\\",
+            "file:///D:/Users/AB025F5/example_dependency_table",
+        ),
+        (
+            "D:\\Users\\AB025F5/example_dependency_table",
+            "file:///D:/Users/AB025F5/example_dependency_table",
         ),
     ),
 )
 def test_ensure_proper_schema_for_local_fs(spark, inputs, outputs) -> None:
     test_fs_utils = FileSystemUtils(spark=spark)
     assert test_fs_utils.ensure_proper_schema_for_local_fs(inputs) == outputs
+
+
+def test_load_and_read_file_from_hadoop(spark, repo_root) -> None:
+    file_path = PurePath(
+        repo_root / "tests/resources/test_common.conf"
+    ).as_posix()
+    config_string = FileSystemUtils(
+        spark=spark
+    ).load_and_read_file_from_hadoop(file_path)
+    assert config_string == (
+        "pramen {\n"
+        '  environment.name = "TestEnv"\n'
+        "  bookkeeping.enabled = true\n"
+        "  bookkeeping.jdbc {\n"
+        '    driver = "org.postgresql.Driver"\n'
+        '    url = "jdbc:postgresql://oss-vip-1973.corp.dsarena.com:5433/syncwatcher_newpipeline"\n'
+        '    user = "app_account"\n'
+        "  }\n"
+        "  warn.throughput.rps = 2000\n"
+        "  good.throughput.rps = 40000\n"
+        "}\n"
+        "mail {\n"
+        '  smtp.host = "smtp.absa.co.za"\n'
+        '  send.from = "Pramen <pramen.noreply@absa.africa>"\n'
+        '  send.to = "ruslan.iushchenko@absa.africa, Dennis.Chu@absa.africa"\n'
+        "}\n"
+        'sourcing.base.path = "/projects/lbamodels/datasync/batch/source"\n'
+    )
+
+
+def test_load_hocon_config_from_hadoop(spark, repo_root) -> None:
+    file_path = PurePath(
+        repo_root / "tests/resources/test_metastore.conf"
+    ).as_posix()
+    hocon_config = FileSystemUtils(spark=spark).load_hocon_config_from_hadoop(
+        file_path
+    )
+    assert hocon_config == ConfigTree(
```

Comment:
This assert expression is very big.
Can we just check if a couple of paths exist in the config? Should be sufficient.
This is because we are not testing if `pyhocon` does its job. We believe it :) But we are testing if a config from hadoop can actually end up parsed.